src/backend/.config/.env.local

```diff
@@ -13,3 +13,20 @@ POSTGRES_PORT = 5432
 POSTGRES_DB = dpia
 POSTGRES_USER = postgres
 POSTGRES_PASSWORD = postgres
+
+################### keycloak setting ###############
+KEYCLOAK_AUTH_SERVER_URI=https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/auth
+KEYCLOAK_RESPONSE_TYPE=code
+KEYCLOAK_SCOPE="openid idir"
+KEYCLOAK_REDIRECT_URI=https://dpia-dev.apps.silver.devops.gov.bc.ca
+KEYCLOAK_CLIENT_ID=
+KEYCLOAK_REALM=standard
+KEYCLOAK_CONFIDENTIAL_PORT
+KEYCLOAK_SSL_REQUIRED=true
+KEYCLOAK_RESOURCE
```

Comment:
What's this?

Reply:
it is the config object used by nestjs keycolak-conect . the same structure as installation.json